packages/xstate-fsm/src/index.ts

```diff
@@ -141,6 +148,7 @@ export function createMachine<
 
             return {
               value: target,
+              config: nextStateConfig,
```

Comment:
shouldn't at least this be easily retrievable by `value` from the config?

Reply:
Agreed, especially because there will be some confusion between this `config` property, and XState (core) `state.configuration`, which is the configuration (set) of state nodes active for that state. Can we remove this?

Reply:
Done. I reverted my 2nd commit, so now it's just the config on the machine.